### PR TITLE
Add .inc grammar generation to Android.mk.

### PR DESCRIPTION
**Pull request by dekimir**
@dneto0 @AWoloszyn @antiagainst @Qining PTAL

### CODE REVIEW

File: third_party/Android.mk [hunk 1]
```diff
@@ -102,24 +102,32 @@ LOCAL_PATH := $(SPVTOOLS_LOCAL_PATH)
...
-$(SPVTOOLS_LOCAL_PATH)/source/operand.cpp: $(1)/operand.kinds.inc
-$(SPVTOOLS_LOCAL_PATH)/source/ext_inst.cpp: $(1)/glsl.std.450.insts.inc $(1)/opencl.std.insts.inc
+$(1)/core.insts-1-1.inc $(1)/operand.kinds-1-1.inc: \
+        $(SPVTOOLS_LOCAL_PATH)/utils/generate_grammar_tables.py \
+        $(SPVTOOLS_LOCAL_PATH)/source/spirv-1-1.core.grammar.json
+		@$(HOST_PYTHON) $(SPVTOOLS_LOCAL_PATH)/utils/generate_grammar_tables.py \
+		                --spirv-core-grammar=$(SPVTOOLS_LOCAL_PATH)/source/spirv-1-1.core.grammar.json \
+		                --core-insts-output=$(1)/core.insts-1-1.inc \
+		                --operand-kinds-output=$(1)/operand.kinds-1-1.inc
+		@echo "[$(TARGET_ARCH_ABI)] Grammar        : instructions & operands <= grammar JSON files"
```
Comment (dneto0) [inline third_party/Android.mk]: Should say 1.1  somehow to differentiate this message from the other message emitted.

**Comment by antiagainst** [comment]
+1

**Comment by dekimir** [comment]
Addressed feedback so far.

**Comment by dneto0** [comment]
+2

**Pull request merged**